### DISCUSSION TITLE
Downloading the data

### DISCUSSION THREAD

**Discussion by akashpujara** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#25] [3.0 VMs and data (Azure VM, local deployment, and Jupyter)]
Hello, can anybody help on how to download the dataset?

**Comment by patrickmuston1** [comment]
Hi akashpujara, the easiest way to get the data is to deploy the docker image to azure using the instructions [here ](https://github.com/EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge/wiki/1b.-Getting-started:-Azure-Environment)and then query the datacube. We haven't made the raw raster files available for download otherwise, however the shapefiles are also [here](https://github.com/EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge/tree/main/notebooks/03_EY_challenge1/resources). Is that what you were after?

**Reply by EY-glowglowglow** [reply]
@akashpujara Hi there! Just wondering if you were able to download the dataset and get started on your analysis? :)

**Comment by akashpujara** [comment]
Hello,

Thank you for your email. I am still learning certain algorithms. Also, am
on my path to learn Azure.

Will complete the same in order to complete the data challenge.

Regards,
Akash Pujara
